build.gradle

```diff
@@ -26,7 +26,8 @@ apply plugin: "net.ltgt.apt"
 
 group = "com.google.api"
 archivesBaseName = "gax"
-version = "0.0.28-SNAPSHOT"
+
+project.version = new File("version.txt").text.trim()
```

Comment:
Is anyone else reading `version.txt`?
If not I think we should just hardcode the content into `build.gradle`